Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocRowRenderer.cs

```diff
@@ -252,7 +252,7 @@ private IBocIndexColumnRenderer GetIndexColumnRenderer ()
       renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Div);
 
       var columnIndexProvider = new BocListColumnIndexProvider(columnRenderers);
-      var arguments = new BocListValidationSummaryRenderArguments(columnIndexProvider ,validationFailuresWithLocationInfo, rowIndex);
+      var arguments = new BocListValidationSummaryRenderArguments(columnIndexProvider ,validationFailuresWithLocationInfo, rowIndex, true);
```

Comment:
please add the parameter name here and in the second renderer. makes reviews and code without Rider-hints easier to check. also, less chance of later mucking up the parameter list when there's more than one boolean.